components/governance/ProposalHeader.vue

```diff
@@ -235,4 +235,10 @@ h2 {
     padding: 2rem 2rem;
   }
 }
+@media (max-width: 768px) {
+  .copy-button {
+    position: absolute;
+    top: 201px;
+  }
+}
```

Comment:
The copy button is one of the child of .header-top-row, which is a flex container, the solution to the problem should be letting the flex elements to wrap, simply adding flex-wrap: wrap is enough